payjoin-cli/src/app/v2/mod.rs

```diff
@@ -363,36 +363,57 @@ impl AppTrait for App {
         self.db.get_inactive_send_session_ids()?.into_iter().try_for_each(
             |(session_id, completed_at)| {
                 let persister = SenderPersister::from_id(self.db.clone(), session_id.clone());
-                if let Ok((sender_state, session_history)) = replay_sender_event_log(&persister) {
-                    let row = SessionHistoryRow {
-                        session_id,
-                        role: Role::Sender,
-                        status: sender_state,
-                        completed_at: Some(completed_at),
-                        error_message: session_history.terminal_error(),
-                    };
-                    send_rows.push(row);
-                }
+                match replay_sender_event_log(&persister) {
```

Comment:
When would replay fail? I think the only exception is expiration. In which case should expiration be a `SessionOutcome` variant instead of returning an error?